### PR TITLE
Grow 28 docker docs

### PR DESCRIPTION
**Pull request by mainocean**
add redme >??????

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,136 @@
...
-```
\ No newline at end of file
+# backend
+
+```python
+python3 -m venv env
+source env/bin/activate
+```
+
+# Start python
```
Comment (dimaserbenyuk) [inline README.md:8]: Create dir /docs
docs 
 - docker .md
 - aws.md

File: README.md [hunk 2]
```diff
@@ -1,28 +1,136 @@
...
+```
+
+# Install DBeaver
+```
+curl https://dbeaver.io/download/
+or
+Snap (sudo snap install dbeaver-ce)
+Flatpak (flatpak install flathub io.dbeaver.DBeaverCommunity)
+```
+# Install PostgreSQL
```
Comment (dimaserbenyuk) [inline README.md:73]: To run all the necessary services we will use docker, the tool can be described in the confltens https://testdemetr.atlassian.net/wiki/home, in the repository we describe only what relates to the code

File: README.md [hunk 3]
```diff
@@ -1,28 +1,136 @@
...
+Bash
+sudo systemctl start postgresql
+Використовуйте цей код обачно.
+
+Accessing PostgreSQL:
+
+Bash
+sudo -i -u postgres
+psql
+Використовуйте цей код обачно.
```
Comment (dimaserbenyuk) [inline README.md:95]: use English ,Since may join different companies and people who don't know Ukrainian

File: README.md [hunk 4]
```diff
@@ -1,28 +1,136 @@
...
+
+Starting PostgreSQL:
+
+Bash
+sudo systemctl start postgresql-14
+sudo systemctl enable postgresql-14
+Використовуйте цей код обачно.
+
+Accessing PostgreSQL:
+
```
Comment (dimaserbenyuk) [inline README.md:118]: Screenshot 2024-07-31 at 13 30 19

File: README.md [hunk 5]
```diff
@@ -1,28 +1,136 @@
...
-
-```python
-pip freeze > requirements.txt
-```
-
-```update
-...
-```
\ No newline at end of file
+# backend
```
Comment (dimaserbenyuk) [inline README.md:1]: What is the goal, to make it convenient to start the whole project in one command and not to install anything, you can leave dbeaver.

**Comment by mainocean** [comment]
add to readme upd  
Screenshot from 2024-08-05 16-07-29

**Pull request closed without merging**